cypress/integration/endEarly_spec.js

```diff
@@ -0,0 +1,26 @@
+describe('end round early', () => {
```

Comment:
Thank you for separating this test since we discuss that it was causing us issues and this seemed to resolve it